src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,41 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private int size = 10;
```

Comment:
```suggestion
    private static final int MAX_SIZE = 10;
```
це краще зробити константою з назвою MAX_SIZE